app/scripts/controllers/landingPage.js

```diff
@@ -84,11 +86,16 @@ angular.module('openshiftConsole')
       }
     }
 
-    function resetRouteParams() {
-      // Reset the route params in the next digest cycle
-      $timeout(function() {
-        $location.replace();
-        $location.search('startTour', null);
-      });
-    }
+    var deregisterLocationStart = $scope.$on('$locationChangeStart', function(event, newUrl) {
+      if (newUrl.indexOf('startTour') > 0) {
+        event.preventDefault();
+        if (tourEnabled) {
+          $scope.startGuidedTour();
+        }
+      }
+    });
+
+    $scope.$on('destroy', function() {
+      deregisterLocationStart();
```

Comment:
I don't think this is needed for `$scope.$on` listeners. They'll be cleaned up automatically. (Although it's required for `$rootScope.$on`)